src/manager/agents/init-module/utils/sc_component_manager_command_init.cpp

```diff
@@ -72,8 +73,16 @@ bool ScComponentManagerCommandInit::ProcessRepositories(ScMemoryContext * contex
     SC_LOG_DEBUG(exception.Message());
   }
 
+  ScAddr component;
   for (ScAddr const & componentSpecificationAddr : currentComponentsSpecificationsAddrs)
   {
+    // TODO componentSpecificationAddr as specification of component
+    component = common_utils::CommonUtils::GetComponentBySpecification(*context, componentSpecificationAddr);
+    if (common_utils::CommonUtils::CheckIfInstalled(*context, component))
+    {
+      SC_LOG_DEBUG("Component \"" + context->HelperGetSystemIdtf(component) + "\" is already installed");
```

Comment:
```suggestion
      SC_LOG_DEBUG("Component \"" << context->HelperGetSystemIdtf(component) << "\" is already installed");
```
Use only << to concat in SC_LOG macros. Check everywhere